### ISSUE TITLE
[Suggestion] log deleted messages feature

### ISSUE THREAD

**Issue by iDaark** [appu1232/Discord-Selfbot#354]
An additional feature to Keyword Logger that logs deleted messages in enabled servers and send them in the same way the keyword logger would do (webhook/private dm)

**Comment by TotalLegitREMIX** [comment]
In addition, the ability to use the "Log History" command in a different channel would be a godsend. If I wanna read deleted messages I would love to be able to pop that lo into my own server or at least into the bot commands channel of the server in question, so as to not clog general (or wherever).

**Comment by LyricLy** [comment]
@TotalLegitREMIX You can with message redirection.

**Comment by TotalLegitREMIX** [comment]
@LyricLy Could you please elaborate?

That is in the section of "I know what I'm doing" and I am about at the "I know some of what I am doing". I input that command into the channel I want, and it gives confirmation "Successfully redirected STDOUT and STDERR to the current channel!" but then nothing gets redirected to that channel. No command responses in the same server or a different server end up there. I tried using "redirect [command]" to test if that does anything, but I still get the same results. 
Does that only redirect cogs? If so could I just make a cog identical to the command I want (IE log history) and use that?

**Comment by Lingrah** [comment]
Okay well this kinda worked, if you need any additional information about this to help resolve this please let me know.

First, after some commands it works correctly, but then also throws an error.
Example 1: `>about >> <channel_id>` it outputs the "about information" but also `An error occurred with the about command. Check the console for details.` command log just has [this](https://i.imgur.com/uNqHMW4.png). 

So the commands work but it ALSO throws an error,

If I run a channel relevant command like `selfdestruct` or `d` then I just get the same check console errors, and the [console looks as follows](https://i.imgur.com/THgo8Gd.png).

When attempting to quote something from channel A, back to that channel A, while running the command in channel B, [I get the following error](https://i.imgur.com/nM4vODq.png).

Lately a 2 parter channel reliant command, like `log history`(the whole reason we are here). Just gives an error(sometimes, of course it wouldn't when I was trying to get screenshots for you), and logs the history of the channel you sent it to, not the channel you sent the original message in.
[Channel in when using command ](https://i.imgur.com/6HuXNMB.png)
[Channel in when confirming command](https://i.imgur.com/RmwHDY8.png)

So as a tl;dr, the redirect command works sometimes, but not in the way you intended most of the time.

**Comment by LyricLy** [comment]
The issue is that when you redirect, it can't find the original command message, so anything trying to affect that will raise an error. I may look into fixing this.

**Comment by clragon** [comment]
@LyricLy yeah, we can redirect the command, but this will still notify people for example in a DM. 
I wrote the suggestion down in my log history improvement issue.
